### PR TITLE
{lib}[gompi/2024a] mpi4py v4.0.0

### PR DESCRIPTION
**Pull request by maxim-masterov**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/mpi4py/mpi4py-4.0.0-gompi-2024a.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+homepage = 'https://github.com/mpi4py/mpi4py'
+description = """MPI for Python (mpi4py) provides bindings of the Message Passing Interface (MPI) standard for
+ the Python programming language, allowing any Python program to exploit multiple processors."""
+
+toolchain = {'name': 'gompi', 'version': '2024a'}
+
+
+dependencies = [
+    ('Python', '3.12.3'),
+    ('Cython', '3.0.10'),
```
Comment (smoors) [inline easybuild/easyconfigs/m/mpi4py/mpi4py-4.0.0-gompi-2024a.eb:15]: Cython should be a build dependency

**Review by smoors — CHANGES_REQUESTED** [review]
please set Cython as builddep?

File: easybuild/easyconfigs/m/mpi4py/mpi4py-4.0.0-gompi-2024a.eb (hunk 1 above)
Comment (maxim-masterov) [reply easybuild/easyconfigs/m/mpi4py/mpi4py-4.0.0-gompi-2024a.eb]: done

**Comment by maxim-masterov** [comment]
> please set Cython as builddep?

done

**Comment by maxim-masterov** [comment]
Test report by @maxim-masterov
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
tcn1.local.snellius.surf.nl - Linux RHEL 9.4, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.9.18
See https://gist.github.com/maxim-masterov/0ddd41e2db2229fae9e99232f59256c3 for a full test report.

**Comment by maxim-masterov** [comment]
@smoors can we go ahead with this one? mpi4py is used as a dependency in some other PRs.

**Comment by smoors** [comment]
@maxim-masterov apparently `mpi4py` v4.0.1 has already been merged  a few days ago.
can you use that version instead?

**Comment by maxim-masterov** [comment]
@smoors ah, I missed it. I updated the relevant PRs to v4.0.1.

**Pull request closed without merging**